modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -81,14 +83,14 @@ $  az vm image list --all --offer rh-ocp-worker --publisher redhat-limited -o ta
 ----
 Offer          Publisher       Sku                 Urn                                                             Version
 -------------  --------------  ------------------  --------------------------------------------------------------  --------------
-rh-ocp-worker  redhat-limited  rh-ocp-worker       redhat-limited:rh-ocp-worker:rh-ocp-worker:4.8.2021122100       4.8.2021122100
-rh-ocp-worker  redhat-limited  rh-ocp-worker-gen1  redhat-limited:rh-ocp-worker:rh-ocp-worker-gen1:4.8.2021122100  4.8.2021122100
+rh-ocp-worker  redhat-limited  rh-ocp-worker       redhat-rhel:rh-ocp-worker:rh-ocp-worker:4.13.2023101100         4.13.2023101100
+rh-ocp-worker  redhat-limited  rh-ocp-worker-gen1  redhat-rhel:rh-ocp-worker:rh-ocp-worker-gen1:4.13.2023101100    4.13.2023101100
 ----
 --
 +
 [NOTE]
 ====
-Regardless of the version of {product-title} that you install, the correct version of the Azure Marketplace image to use is 4.8. If required, your VMs are automatically upgraded as part of the installation process.
+Regardless of the version of {product-title} that you install, the correct version of the Azure Marketplace image to use is 4.13. If required, your VMs are automatically upgraded as part of the installation process.
```

Comment:
I wonder if this version is use anywhere else and if it would be worthwhile to set a variable? It looks like updating it from version 4.9 to 4.12 was missed. I'm not sure the best practice for keeping up to date with these images, though.